### ISSUE TITLE
Check and update node status (e.g., opened, visible) when opening historymap

### ISSUE THREAD

**Issue by Yuu-Han** [Vis4Sense/SenseMapExtension#176]

**Comment by kaidatavis** [comment]
Now historymap would automatically load the previous session and some of the node might be open.

**Comment by Yuu-Han** [comment]
To be consistent with the current session, better not to load node states from previous session.

In case the user wants to restore the previous session, maybe add a button to highlight nodes in the previous session (opened) and let the user decide which nodes to restore?

<img width="681" alt="image" src="https://github.com/Vis4Sense/HistoryMap/assets/72588097/0893785a-3130-466c-aaa1-31432230279c">

**Comment by kaidatavis** [comment]
I agree. Maybe just a button so user can restore all the opened tabs from the previous session? Selecting which tab to open individually would be the same as clicking the nodes in historymap?
